README.md

```diff
@@ -1,17 +1,37 @@
 # Lambda Curry Medusa Forms
 
+<div align="center">
+  
+![Lambda Curry Logo](https://lambdacurry.dev/favicon.ico)
```

Comment:
this is great, but let's also apply some Lambda Curry branding to our https://lambda-curry.github.io/medusa-forms/pr-6/?path=/docs/0-1-hello-world-start-here--docs page as well